server/src/main/java/info/doseamigos/authusers/DefaultAuthUserService.java

```diff
@@ -28,6 +31,24 @@ public DefaultAuthUserService(@Nonnull  AuthUserDao authUserDao) {
         this.authUserDao = requireNonNull(authUserDao);
     }
 
+    @Override
+    public String getAccessToken(String idToken) throws IOException {
+        HttpGet httpGet = new HttpGet("https://mckoon.auth0.com/tokeninfo?id_token=" + idToken);
+        HttpClient client = HttpClients.createDefault();
+        HttpResponse response = client.execute(httpGet);
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        Map<String, Object> idTokenResults = objectMapper.readValue(response.getEntity().getContent(), objectMapper.getTypeFactory()
+            .constructMapType(HashMap.class, String.class, Object.class));
+        String toRet = null;
+        for (Map<String, Object> identity : (List<Map<String, Object>>) idTokenResults.get("identities")) {
```

Comment:
Some comments here might not be a bad thing.

The idea is that the object we get back from Auth0 contains a list of identities that is used.  I specifically search out the google one and grab that access token and validate what google thinks.